pages.pl/common/clang-cpp.md

```diff
@@ -1,6 +1,6 @@
 # clang-cpp
 
-> Te polecenie jest aliasem `clang++`.
+> To poleceniejest aliasem `clang++`.
```

Comment:
Missing space. Should be "polecenie jest".

Reply:
Fixed, sorry :)